firebase-firestore/src/test/java/com/google/firebase/firestore/model/ObjectValueTest.java

```diff
@@ -32,7 +34,19 @@
 
 @RunWith(RobolectricTestRunner.class)
 @Config(manifest = Config.NONE)
-public class FieldValueTest {
+public class ObjectValueTest {
+
+  private String fooString = "foo";
+  private Value fooValue = wrap(fooString);
+  private String barString = "bar";
+  private Value barValue = wrap(barString);
+  private Value emptyObject = Value.newBuilder().setMapValue(MapValue.getDefaultInstance()).build();
+
+  @Test
+  public void supportsEmptyObjectValues() {
```

Comment:
nit: All of the other test methods in this class have their names prefixed with "test". Can you rename `supportsEmptyObjectValues` to also follow this convention?

Reply:
Done